### PR TITLE
[tests-only][full-ci]added bug demo test for removing systemtags app from whitelist for guest users

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR adds bug demonstration test for removing `systemtag` app from whilelist for guest users, so that guest user cannot add, remove or view tags of a resource.

## Related Issue
Related to issue: https://github.com/owncloud/guests/issues/551

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test added

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by grgprarup — COMMENTED** [review]
lgtm

File: tests/acceptance/features/apiGuests/guests.feature [hunk 1]
```diff
@@ -431,3 +431,50 @@ Feature: Guests
     And user "Alice" should have the following comments on file "/textfile0.txt"
       | user  | comment          |
       | Alice | My first comment |
+
+  @email @issue-551
+  Scenario: A guest user cannot add tags on resource when systemtags app is not whitelisted
```
Comment (grgprarup) [inline tests/acceptance/features/apiGuests/guests.feature:436]:
```suggestion
  Scenario: a guest user cannot add tags on resource when systemtags app is not whitelisted
```

File: tests/acceptance/features/apiGuests/guests.feature [hunk 2]
```diff
@@ -431,3 +431,50 @@ Feature: Guests
...
+    And guest user "guest" has registered
+    And the administrator has limited the guest access to the default whitelist apps
+    And the administrator has removed the app "systemtags" from the whitelist for the guest user
+    When user "guest@example.com" adds tag "MyFirstTag" to file "/textfile0.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    # uncomment the line below and remove the line above when issue-551 is fixed
+    #Then the HTTP status code should be "403"
+
+  @email @issue-551
+  Scenario: A guest user cannot view tags of resource when systemtags app is not whitelisted
```
Comment (grgprarup) [inline tests/acceptance/features/apiGuests/guests.feature:451]:
```suggestion
  Scenario: a guest user cannot view tags of resource when systemtags app is not whitelisted
```

File: tests/acceptance/features/apiGuests/guests.feature [hunk 3]
```diff
@@ -431,3 +431,50 @@ Feature: Guests
...
+    And guest user "guest" has registered
+    And the administrator has limited the guest access to the default whitelist apps
+    And the administrator has removed the app "systemtags" from the whitelist for the guest user
+    When user "guest@example.com" requests tags for file "/textfile0.txt" owned by user "Alice" using the WebDAV API
+    Then the HTTP status code should be "207"
+    # uncomment the line below and remove the line above when issue-551 is fixed
+    #Then the HTTP status code should be "403"
+
+  @email @issue-551
+  Scenario: A guest user cannot remove tags of resource when systemtags app is not whitelisted
```
Comment (grgprarup) [inline tests/acceptance/features/apiGuests/guests.feature:467]:
```suggestion
  Scenario: a guest user cannot remove tags of resource when systemtags app is not whitelisted
```

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by phil-davis — APPROVED** [review]
LGTM. During the development of the fix, the developer and QA etc can sort out if these system-tag requests by the guest should return 403 (forbidden) or 404 (not found), and adjust the scenario steps as needed.

**Pull request merged**